src/main/java/Dictionary/Word.java

```diff
@@ -0,0 +1,76 @@
+package Dictionary;
+
+
+import java.util.HashSet;
+
+/**
+ * Represents words stored in the Word Bank
+ */
+
+public class Word {
+    private String word;
+    private String meaning;
+    private HashSet<String> tags;
+
+    /**
+     * Number of times that a word is searched
+     */
+    private int numberOfSearches;
+
+    /**
+     * String represents the closest time that user search
+     * for a specific word
+     */
+    private String closetSearch;
```

Comment:
closestSearch you mean?
To clarify, closest time -> this is a string containing a time format?